test/core/crypto/crypto_store/crypto_store_test.cpp

```diff
@@ -293,17 +293,36 @@ TEST_F(CryptoStoreTest, getEd25519PublicKeysSuccess_DISABLED) {
   EXPECT_OUTCOME_TRUE(pair4, crypto_store->generateSr25519Keypair(kBabe));
   EXPECT_OUTCOME_TRUE(pair5, crypto_store->generateSr25519Keypair(kAcco));
 
-  CryptoStoreImpl::ED25519Keys ed_babe_keys = {pair1.public_key,
-                                               pair2.public_key};
+  std::set<ED25519PublicKey> ed_babe_keys_set = {pair1.public_key,
+                                                 pair2.public_key};
+  std::vector<ED25519PublicKey> ed_babe_keys(ed_babe_keys_set.begin(),
+                                             ed_babe_keys_set.end());
```

Comment:
What is std::set for? Seems it redundant and we should fill vector directly.

Reply:
The order is important, if a container was initialized from set, it has have sorted items.